cartographer/resources/resource_registry.py

```diff
@@ -32,6 +33,9 @@ def call_initialization_hook(self):
     def register_resource(self, type_string, schema,
                           serializer=None, parser=None, mask=None,
                           model=None, model_get=None, model_prime=None,):
+        if model and serializer:
+            self.serializers[model] = serializer
```

Comment:
How would that work? What's the logic behind it? why is the param not called `serializers` to reflect that? is there an example for how/where it's used?